.github/CONTRIBUTING.md

```diff
@@ -1,5 +1,6 @@
 # Contributing to DESOSA 2017
 
+To be able to contribute to this repository, fork this repository on your personal account and submit a pull request on your fork.
```

Comment:
I'd rephrase this to something like 'submit a pull request from your fork'. Submitting a pull request _on_ your fork sounds a little ambiguous to me.